__init__.py

```diff
@@ -1,6 +1,6 @@
 import sys
 import os.path
 
-# Add /testing/ to PYTHONPATH:
+# Add /testing/ directory to PYTHONPATH:
```

Comment:
We could drop this change as we should still trigger the *build* of the docker image for the Check job: https://teamcity.mesosphere.io/admin/editTriggers.html?id=buildType:DcosIo_DcosCommons_PrJobs_RunCodeChecks which is what was causing the earlier failures.